_studio/mfx_lib/decode/vp9/include/mfx_vp9_dec_decode_hw.h

```diff
@@ -37,6 +37,8 @@
 
 namespace UMC_VP9_DECODER { class Packer; }
 
+class ReferenceFrameStorage;
```

Comment:
The class name is confused. It is not Reference frames it is frames submitted to the driver or frames in decoding. My concern is VP9DecoderFrame structure contains ref_frame_map  which is the list of reference frames.